backend/homework/1/core/src/main/java/org/handson/Student.java

```diff
@@ -0,0 +1,43 @@
+package org.handson;
+
+public class Student
+{
+int id;
+String name;
+int age;
+char grade;
+public void setID(int ID)
+{
+    id=ID;
+}
+public void setName(String Name)
+{
```

Comment:
use `this.name = Name` to make sure that the parameter value is assigned to the instance variable